gradle-plugin/it/src/it/git-annotations/expected/kubernetes.yml

```diff
@@ -6,7 +6,7 @@ items:
   kind: Deployment
   metadata:
     annotations:
-      jkube.eclipse.org/git-url: "@matches('(?:git|ssh|https?|git@[-\\w.]+):(\\/\\/)?(.*?)(\\.git)(\\/?|\\#[-\\d\\w._]+?)$')@"
+      jkube.eclipse.org/git-url: "@matches('(?:git|ssh|https?|git[-\\w.]+):(\\/\\/)?(.*?)(\\.git)(\\/?|\\#[-\\d\\w._]+?)$')@"
```

Comment:
I don't think this is OK, this probably passes because the remote url starts with `git`_hub.com_`:`

Since there aren't that many cases for this specific scenario, maybe we can just simplify the regex to match for https and git URLs.